config_spec.toml

```diff
@@ -0,0 +1,19 @@
+[[param]]
+name = "address"
+type = "String"
+optional = false
+doc = "LND GRPC address, kindly note that the address must start with https:// ."
+
+
+[[param]]
+name = "cert"
+type = "String"
```

Comment:
This should be `std::path::PathBuf` so that people using uncommon encoding will not get nasty surprises.

Reply:
I aggree :100: